tests/checks/value_length.jinja

```diff
@@ -0,0 +1,23 @@
+{% macro value_length(columns, expected_length, where) %}
+  {% if columns is string %}
```

Comment:
do you really need a list of columns? and if so wouldn't you also want to list the values?

`value_length([user_id, country_code], [36, 2])` 

I would prefer not allowing lists at all so you have to specify this rule explicitly for each col though

Reply:
Made the change, actually just simplified the whole check.